.github/CODEOWNERS

```
@@ -478,6 +478,12 @@
 # ServiceLabel: %TimeSeriesInsights
 # ServiceOwners:                                                   @yeskarthik @rasidhan @dmdenmsft
 
+# PRLabel: %Defender EASM
+/sdk/easm/Azure.Analytics.Defender.Easm                            @thang-bit
```

```suggestion
/sdk/easm/Azure.Analytics.Defender.Easm/                           @thang-bit
```

The path must end with a slash.